content/en/opentelemetry/interoperability/environment_variable_support.md

```diff
@@ -0,0 +1,155 @@
+      tag: 'Documentation'
+      text: 'Go SDK Configuration'
+    - link: '/tracing/trace_collection/library_config/java'
+      tag: 'Documentation'
+      text: 'Java SDK Configuration'
+    - link: '/tracing/trace_collection/library_config/nodejs'
+      tag: 'Documentation'
+      text: 'Node.js SDK Configuration'
+    - link: '/tracing/trace_collection/library_config/php'
+      tag: 'Documentation'
+      text: 'PHP SDK Configuration'
+    - link: '/tracing/trace_collection/library_config/python'
+      tag: 'Documentation'
+      text: 'Python SDK Configuration'
+    - link: '/tracing/trace_collection/library_config/ruby'
+      tag: 'Documentation'
+      text: 'Ruby SDK Configuration'
+---
+
+Datadog SDKs support the following OpenTelemetry configuration options, allowing you to use OpenTelemetry environment variables to configure Datadog tracing for your applications.
+
+<div class="alert alert-info">If both Datadog and OpenTelemetry environment variables are set, Datadog takes precedence. Datadog defaults also override OpenTelemetry defaults. See the relevant <a href="/tracing/trace_collection/library_config/">SDK Configuration page</a> for default values and more information.</div>
+
+## General SDK configuration
+Datadog SDKs support the following general OpenTelemetry SDK options. For more information, see the related [OpenTelemetry documentation][9].
+
+`OTEL_SERVICE_NAME`
+: ****Datadog convention****: `DD_SERVICE`<br>
+Sets the service name<br>
+**Notes**: If `service.name` is also provided in `OTEL_RESOURCE_ATTRIBUTES`, then `OTEL_SERVICE_NAME` takes precedence<br>
+
+`OTEL_LOG_LEVEL`
+: ****Datadog convention****: `DD_LOG_LEVEL`<br>
+Log level used by the SDK logger<br>
+**Notes**: A log level of debug will also map to `DD_TRACE_DEBUG=true`<br>
+In the Node.js & PHP SDKs this maps to `DD_TRACE_LOG_LEVEL` <br>
+In the Go SDK only mapped values between `OTEL_LOG_LEVEL` & `DD_TRACE_DEBUG` are supported:<br>
+  - `info`|`false`
+  - `debug`|`true`
+**Not Supported In**: Python, .NET, Ruby, and Go SDKs<br>
+
+`OTEL_PROPAGATORS`
+: ****Datadog convention****: `DD_TRACE_PROPAGATION_STYLE`<br>
+Propagators to be used as a comma-separated list<br>
+**Notes**: the only supported values for most Datadog SDKs are `tracecontext`, `b3`, `b3multi`, `none`, `datadog`. `xray` is also supported for the Java SDK<br>
+Values MUST be deduplicated in order to register a `Propagator` only once<br>
+
+`OTEL_TRACES_SAMPLER & OTEL_TRACES_SAMPLER_ARG`
+: ****Datadog convention****: `DD_TRACE_SAMPLE_RATE`<br>
+`OTEL_TRACES_SAMPLER`: Sampler to be used for traces & `OTEL_TRACES_SAMPLER_ARG`: String value to be used as the sampler argument<br>
+**Notes**: The specified value will only be used if `OTEL_TRACES_SAMPLER` is set. Each Sampler type defines its own expected input, if any. Invalid or unrecognized input MUST be logged and MUST be otherwise ignored, i.e. the implementation MUST behave as if `OTEL_TRACES_SAMPLER_ARG` is not set<br>
```

Comment:
```suggestion
**Notes**: The specified value is only used if `OTEL_TRACES_SAMPLER` is set. Each Sampler type defines its own expected input, if any. Invalid or unrecognized input MUST be logged and MUST be otherwise ignored. In such cases, the implementation MUST behave as if `OTEL_TRACES_SAMPLER_ARG` is not set<br>
```